lambdas/data-migration1/tests/test-collections.js

```diff
@@ -101,16 +101,20 @@ test.serial('migrateCollectionRecord correctly migrates collection record', asyn
     .first();
 
   t.deepEqual(
-    omit(createdRecord, ['cumulusId']),
+    omit(createdRecord, ['cumulus_id']),
     omit(
       {
         ...fakeCollection,
-        granuleIdValidationRegex: fakeCollection.granuleId,
-        granuleIdExtractionRegex: fakeCollection.granuleIdExtraction,
+        sample_file_name: fakeCollection.sampleFileName,
+        granule_id_validation_regex: fakeCollection.granuleId,
+        granule_id_extraction_regex: fakeCollection.granuleIdExtraction,
+        duplicate_handling: fakeCollection.duplicateHandling,
+        report_to_ems: fakeCollection.reportToEms,
+        ignore_files_config_for_discovery: fakeCollection.ignoreFilesConfigForDiscovery,
         created_at: new Date(fakeCollection.createdAt),
         updated_at: new Date(fakeCollection.updatedAt),
       },
-      ['granuleId', 'granuleIdExtraction', 'createdAt', 'updatedAt']
+      ['granuleId', 'granuleIdExtraction', 'sampleFileName', 'granuleIdValidationRegex', 'granuleIdExtractionRegex', 'ignoreFilesConfigForDiscovery', 'duplicateHandling', 'reportToEms', 'createdAt', 'updatedAt']
```

Comment:
This (and the others like it ) feels like something that should be consolidated as a test-suite structure, rather than repeated in each test.